tests/conftest.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+import os
+from astrodbkit2.astrodb import create_database, Database
+from astrodb_scripts import load_astrodb
+from simple import REFERENCE_TABLES
+from simple.schema import *
+
+# Create a fresh temporary SIMPLE database and assert it exists
+# Because we've imported simple.schema, we will be using that schema for the database
+# This is in the conftest.py file so temp database is only created once per test session
```

Comment:
I haven't used conftest files much, but we should probably not have any `assert` calls if this is just meant to be fixtures.